src/Firehose.Web/Infrastructure/NewCombinedFeedSource.cs

```diff
@@ -182,6 +182,27 @@ DateTimeOffset GetMaxTime(SyndicationItem item)
 				LastUpdatedTime = DateTimeOffset.UtcNow
             };
 
+            // This foreach section is responsible for adding the Twitter handles
+            foreach (var item in feed.Items)
+            {
+                string newTitleText = string.Empty;
+                if (item.Authors.Count > 0)
+                    newTitleText = Tamarins.Where(x => x.FirstName + " " + x.LastName == item.Authors[0].Name).FirstOrDefault()?.TwitterHandle ?? string.Empty;
+                newTitleText = String.IsNullOrEmpty(newTitleText) ? item.Title.Text : item.Title.Text + " @" + newTitleText;
+
+                if (!string.IsNullOrWhiteSpace(item.Title.Type))
+                {
+                    string type = item.Title.Type == "text" ? "Plaintext" : item.Title.Type;
+                    TextSyndicationContentKind textKind = (TextSyndicationContentKind)
+                        Enum.Parse(typeof(TextSyndicationContentKind), type, ignoreCase: true);
+                    item.Title = new TextSyndicationContent(newTitleText, textKind);
+                }
+                else
+                {
+                    item.Title = new TextSyndicationContent(newTitleText);
+                }
```

Comment:
This improves readability a bit. Fitting everything into `newTitleText` is a bit of a smell to me.

Also, I prefer not to use ternaries if possible.

```suggestion
                var twitterHandle = string.Empty;
                if (item.Authors.Count > 0)
                {
                    twitterHandle = Tamarins.FirstOrDefault(x =>
                        $"{x.FirstName} {x.LastName}" == item.Authors[0].Name)?.TwitterHandle ?? string.Empty;
                }

                var title = item.Title.Text;
                if (!string.IsNullOrEmpty(twitterHandle))
                    title += $"@{twitterHandle}";

                if (!string.IsNullOrWhiteSpace(item.Title.Type))
                {
                    string type = item.Title.Type == "text" ? "Plaintext" : item.Title.Type;
                    var textKind = (TextSyndicationContentKind)
                        Enum.Parse(typeof(TextSyndicationContentKind), type, ignoreCase: true);
                    item.Title = new TextSyndicationContent(title, textKind);
                }
                else
                {
                    item.Title = new TextSyndicationContent(title);
                }
```

Reply:
Removed newTitleText and got rid of ternaries